js/gate.js

```diff
@@ -210,6 +212,26 @@ module.exports = class gate extends Exchange {
                             'sub_account_transfers': 300,
                         },
                     },
+                    'sub_accounts': {
```

Comment:
@sc0Vu why not do something like this?

```Javascript

(...)
'subAccounts': 'https://api.gateio.ws/api/v4'
(,,,)

'subAccounts': {
     'get': {
            'sub_accounts':1,
            'sub_accounts/{user_id}:1,
             (....)
     }
}

```
I think that by doing this ⬆️ you don't have to change anything, and we avoid a weird format, lmk what you think

Reply:
@carlosmiei you're right, I also consider about this:

```
(...)
'subAccounts': 'https://api.gateio.ws/api/v4/sub_accounts'
(...)

'subAccounts': {
     'get': {
            '':1,
            '{user_id}':1,
             (....)
     }
}
```

In this way, user don't need to type another SubAccounts.

Reply:
I understand your point, but I'd rather have to duplicate the path a bit than have an empty key, what do you think? @kroitor

Reply:
@carlosmiei  I removed the updates in Exchange.js.